src/late_multicellular_stage/SCALIS.cs

```diff
@@ -0,0 +1,272 @@
+﻿using System;
+using System.Collections.Generic;
+using Godot;
+
+public class Scalis : IMeshGeneratingFunction
```

Comment:
Filename should match exactly the name of the class.

Reply:
Also would be pretty nice to have a summary comment on this. I won't ask for detailed comments for everything in this file as this is probably some advanced math stuff but a summary would be nice of what this does.